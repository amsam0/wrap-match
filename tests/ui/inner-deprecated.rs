#[deny(deprecated)]
fn main() {
    InnerDeprecated._wrap_match_inner_test().unwrap_err();
}

struct InnerDeprecated;

impl InnerDeprecated {
    #[wrap_match::wrap_match]
    pub fn test(&self) -> Result<(), ()> {
        Err(())?;
        Ok(())
    }
}
