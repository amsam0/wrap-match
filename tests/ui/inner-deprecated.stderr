error: use of deprecated method `InnerDeprecated::_wrap_match_inner_test`: inner function for wrap-match. Please do not use!
 --> tests/ui/inner-deprecated.rs:3:21
  |
3 |     InnerDeprecated._wrap_match_inner_test().unwrap_err();
  |                     ^^^^^^^^^^^^^^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/ui/inner-deprecated.rs:1:8
  |
1 | #[deny(deprecated)]
  |        ^^^^^^^^^^
